src/figuresClasses.ts

```diff
@@ -2,10 +1,88 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('error message');
```

Comment:
The error message thrown here is not descriptive. It should explain what went wrong, such as 'Triangle side lengths must be greater than zero.'